cpp/CMakeLists.txt

```diff
@@ -121,6 +126,11 @@ set_target_properties(
              INTERFACE_POSITION_INDEPENDENT_CODE ON
 )
 
+# Need to add this define, as is done in legate CMakeLists.txt as well for __half support.  If
+# __half support fails elsewhere it may be needed there. This may be a CCCL 2.7.0 issue and become
+# unnecessary in the future.
+target_compile_definitions("LegateDataframe" PUBLIC _LIBCUDACXX_HAS_NVFP16=1)
```

Comment:
@RAMitchell this is something to remember when doing this for legate-boost. (Also the CCCL thing likely, while it makes sense here, it is a bit surprising that it picked up a random CCCL version rather than one from legate, since I think legate was the first dependency here.)
The CCCL was a bit confusing, because in some versions even when it claimed to use 2.7, it wasn't.

Reply:
Noted thanks!